server/index.js

```diff
@@ -14,8 +14,8 @@ app.post('/recipe', (req, res) => {
 })
 
 
-app.get('/recipe', (req, res) => {
-  return Comments.getRecipe()
+app.get('/recipe/:id', (req, res) => {
+  return Comments.getComment(req.params.id)
```

Comment:
we aren't returning anything here; getComment is an asynchronous function and the information we need to send back to the client is information you receive after the promise resolves.

Reply:
Removed return as there is no need for it